components/rich_text_editor/extensions/emoji/EmojiList.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    v-show="items.length"
+    class="dt-emoji-suggestion-list"
+  >
+    <dt-list-item
+      v-for="(item, index) in items"
+      :key="index"
+      :class="[
+        'dt-emoji-suggestion-list-item',
+        { 'is-selected': index === selectedIndex },
+      ]"
+      navigation-type="arrow-keys"
+      @click="selectItem(index)"
+      @keydown.prevent.stop="onKeyDown"
+    >
+      <dt-emoji
+        size="200"
+        :code="item"
+      />
+      <div class="dt-emoji-suggestion-list-text">
+        {{ item }}
+      </div>
+    </dt-list-item>
+  </div>
+</template>
+
+<script>
+import { DtEmoji } from '@/emoji';
+import { DtListItem } from '@/components/list_item';
+
+export default {
+  name: 'EmojiList',
+  components: {
+    DtEmoji,
+    DtListItem,
+  },
+
+  props: {
+    items: {
+      type: Array,
+      required: true,
+    },
+
+    command: {
+      type: Function,
+      required: true,
+    },
+  },
+
+  data () {
+    return {
+      selectedIndex: 0,
+    };
+  },
+
+  watch: {
+    items () {
+      this.selectedIndex = 0;
+    },
+  },
+
+  methods: {
+    onKeyDown ({ event }) {
+      if (event.key === 'ArrowUp') {
+        this.upHandler();
+        return true;
+      }
+
+      if (event.key === 'ArrowDown') {
+        this.downHandler();
+        return true;
+      }
+
+      if (event.key === 'Enter') {
+        this.enterHandler();
+        return true;
+      }
+
+      return false;
+    },
+
+    upHandler () {
+      this.selectedIndex = ((this.selectedIndex + this.items.length) - 1) % this.items.length;
+    },
+
+    downHandler () {
+      this.selectedIndex = (this.selectedIndex + 1) % this.items.length;
+    },
+
+    enterHandler () {
+      this.selectItem(this.selectedIndex);
+    },
+
+    selectItem (index) {
+      const item = this.items[index];
+
+      if (item) {
+        this.command({ code: item });
+      }
+    },
+  },
+};
+</script>
+
+<style>
+.dt-emoji-suggestion-list {
+  background-color: var(--dt-color-surface-secondary) !important;
+  border-color: var(--dt-color-border-subtle) !important;
+  max-height: 216px !important;
```

Comment:
Should not have `px` or `rem` in recipes styles, you will probably need to use [sizing tokens](https://dialpad.design/tokens/sizing.html )  or [spacing tokens](https://dialpad.design/tokens/spacing.html) with `calc()` to achieve this values. I will ping @juliodialpad just to be sure.

Reply:
Double checked with Julio, yes it is always necessary to use token, you can modified it @sanjeevdialpad

Reply:
changed every value to use tokens.